app/views/layouts/shared/_footer.html.erb

```diff
@@ -76,9 +76,9 @@
         <!-- contact info -->
         <div class="mb-4">
           <h3 class="fw-bold mb-3">Get in touch</h3>
-          <p>339 McDermott Points Hettingerhaven, NV 15283</p>
-          <p class="mb-1">Email: <a href="#">support@geeksui.com</a></p>
-          <p>Phone: <span class="text-dark fw-semibold">(000) 123 456 789</span></p>
+          <p><%= Current.tenant.location.city_town %>, <%= Current.tenant.location.province_state %> <%= Current.tenant.location.country %></p>
+          <p class="mb-1">Email: <a href="mailto:<%= Current.tenant.profile.email %>"><%= Current.tenant.profile.email %></a></p>
+          <p>Phone: <span class="text-dark fw-semibold"><%= Current.tenant.profile.phone_number %></span></p>
```

Comment:
What are your thoughts on formatting the phone number? 

```suggestion
          <p>Phone: <span class="text-dark fw-semibold"><%= Current.tenant.profile.formatted_phone %></span></p>
```

![Screenshot from 2024-05-01 19-36-28](https://github.com/rubyforgood/pet-rescue/assets/16829344/50325ebf-1469-4c3a-906f-562435f66c6d)

We have a method in the `adopter_foster_profile` model you could reference and add to `organization_profile` model

Reply:
Sounds good, I'll take a look and pull it into a concern for reuse

Reply:
Hey @jmilljr24 I went ahead and refactored this into a concern. I included the `before_save` normalizing in there as well. Let me know what you think.